returns/result.py

```diff
@@ -47,14 +50,22 @@ class Result(
 
     """
 
+    __slots__ = ('_trace',)
```

Comment:
Are `__slots__` combined when inheritance is used? (sorry, I don't know how it works)

I am asking because I was expecting to see  `__slots__ = ('_inner_value', '_trace',)`

Reply:
I had the same question! Yes, `__slots__` are combined and if we overlap an attribute in child class it can waste memory!
[How does inheritance of \_\_slots__ in subclasses actually work?](https://stackoverflow.com/questions/1816483/how-does-inheritance-of-slots-in-subclasses-actually-work)

Another point here is overlapping slots attributes can be tricky, take a look at the sixth item:
[Notes on using slots](https://docs.python.org/3/reference/datamodel.html#notes-on-using-slots)

Reply:
TIL 👍